firestore/cloud-client/snippets.py

```diff
@@ -61,7 +61,7 @@ def quickstart_get_collection():
     docs = users_ref.stream()
 
     for doc in docs:
-        print(u'{} => {}'.format(doc.id, doc.to_dict()))
+        print(f'{doc.id} => {doc.to_dict()}')
```

Comment:
Change looks good.
Just a thought. Does it make sense to use [`pprint`](https://docs.python.org/3/library/pprint.html)?

Reply:
Just tried, it's not very pretty unfortunately.
```
>>> pprint(doc)
<google.cloud.firestore_v1.document.DocumentSnapshot object at 0x10dfa1780>
```

Reply:
Thanks, LGTM